packages/frontend-plugin-api/api-report.md

```diff
@@ -356,6 +356,10 @@ export interface BackstagePlugin<
   readonly id: string;
   // (undocumented)
   readonly routes: TRoutes;
+  // (undocumented)
+  withOverrides(options: {
+    extensions: Array<ExtensionDefinition<any, any>>;
+  }): BackstagePlugin<TRoutes, TExternalRoutes, TExtensionMap>;
```

Comment:
Just a note that we may want to make these types smarter to cover for newly added extension IDs